src/commands/emoji.ts

```diff
@@ -0,0 +1,34 @@
+import { DiscordAPIError, EmbedBuilder, SlashCommandBuilder } from 'discord.js';
+import * as logger from '../logger';
+
+const EmojiName = 'emojiname';
+
+const builder = new SlashCommandBuilder()
+	.setName('emoji')
+	.setDescription('Responds with the picture used for the mentioned emoji')
+	.addStringOption(option =>
+		option.setName(EmojiName).setDescription('Name of the emoji to search for').setRequired(true)
+	);
+
+export const emoji: GlobalCommand = {
+	info: builder,
+	requiresGuild: false,
+	async execute({ reply, interaction }): Promise<void> {
+		const emojiName = interaction.options.getString(EmojiName);
+		if (emojiName === null) {
+			throw new Error('You must provide an emoji to search for');
+		}
```

Comment:
Since we mark the option as required in the builder, we can pass `true` as the second arg and we don't have to handle the non-existent case where the value isn't provided!

```suggestion
		const emojiName = interaction.options.getString(EmojiName, true);
```